lib/common/options.js

```diff
@@ -209,6 +209,22 @@ Options.from = function (_options) {
     options.build.extractCSS = false
   }
 
+  const loaders = options.build.loaders
+  const vueLoader = loaders.vue
+  if (vueLoader.productionMode === undefined) {
+    vueLoader.productionMode = !options.dev
+  }
+  const styleLoaders = [
+    'css', 'cssModules', 'less',
+    'sass', 'scss', 'stylus', 'vueStyle'
+  ]
+  for (const name of styleLoaders) {
+    const loader = loaders[name]
+    if (loader) {
```

Comment:
Suggest changing to `if (loader && typeof loader.sourceMap === 'undefined')` so that user can override it

Reply:
I know this is nothing common but the way that we override it makes it impossible :)